lightning/src/ln/features.rs

```diff
@@ -346,6 +348,9 @@ mod sealed {
 	define_feature!(19, Wumbo, [InitContext, NodeContext],
 		"Feature flags for `option_support_large_channel` (aka wumbo channels).", set_wumbo_optional, set_wumbo_required,
 		supports_wumbo, requires_wumbo);
+	define_feature!(23, AnchorsZeroFeeHtlcTx, [InitContext, NodeContext, ChannelTypeContext],
```

Comment:
According to BOLT9, `option_anchors_zero_fee_htlc_tx` is IN, so I don't know if we should be only `InitContext` / `NodeContext` here? The spec feature bit context nomenclature is obscure, without that much ground so I suppose it could also make sense to amend the spec here.

Reply:
Yes, effectively the `channel_type` field is described in BOLT2 without back reference in BOLT9 -- For the confusion of everyone.